### PR TITLE
Update to match latest DWS/NNF

### PR DESCRIPTION
**Pull request by roehrich-hpe**

### CODE REVIEW

**Comment by roehrich-hpe** [comment]
@bdevcich do use this now, you have to update the container profile to change the image tag to your build.

I'm trying to figure out how to get the rabbit node name into the hello-world, so it can build the proper path into the storage. So far, it looks like mpirun does not allow the app to see the environment variables.

**Comment by roehrich-hpe** [comment]
> @bdevcich do use this now, you have to update the container profile to change the image tag to your build.
> 
> I'm trying to figure out how to get the rabbit node name into the hello-world, so it can build the proper path into the storage. So far, it looks like mpirun does not allow the app to see the environment variables.

Well, the sshd on the worker can see the env var, but it doesn't pass its own env to the things that it spawns.
I don't yet know how to tell the mpi process that lands on the rabbit, which rabbit name it's on.

**Comment by bdevcich** [comment]
> I don't yet know how to tell the mpi process that lands on the rabbit, which rabbit name it's on.

Perhaps we need to inject something like this into the pod spec:

```
env:
  - name: NNF_NODE
    valueFrom:
      fieldRef:
        fieldPath: spec.nodeName
```

I'm hoping you could add this to the container profile and see if it works. If it does, then I think we need to make sure that nnf-sos ensure this field is in there.

**Comment by bdevcich** [comment]
OK, it looks like you're already doing something like that, but it doesn't look like the launcher pod has the env field:

```
$kubectl get pods demo-container-launcher-88x5x -oyaml | yq '.spec.containers[].env' | grep NNF_NODE
$
```

You'll need to add that to the Launcher in the container profile as well.

File: nnf-container-example.yaml [hunk 1]
```diff
@@ -34,14 +34,14 @@ data:
...
-                # default. Leaving this here for now in case an older verion of
-                # nnf-sos is used.
-                # securityContext:
-                  # capabilities:
-                    # add: ["NET_BIND_SERVICE", "SYS_CHROOT", "AUDIT_WRITE", "SETUID", "SETGID"]
+                env:
+                  - name: NNF_NODE_NAME
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: spec.nodeName
```
Comment (bdevcich) [inline nnf-container-example.yaml]: This needs to go in the Launcher's spec as well

**Pull request closed without merging**